### DISCUSSION TITLE
GHG emissions for specific livestock products

### DISCUSSION THREAD

**Discussion by FrancoBilotto** [magpiemodel/magpie#432] [Help]
Dear MagPIE community,

I am struggling to dissagregate emissions such as enteric fermentation from different livestock products like beef cattle.
Does anyone know how to make the request within the R code to extract this sort of data?

Please, I need some help here guys!
Cheers,
Franco

**Comment by emolinab** [comment]
Hi Franco! In magpie4 there is a function called [reportEmissions](https://github.com/pik-piam/magpie4/blob/master/R/reportEmissions.R). In R, you call the library (`library(magpie4)`), and then simply use `reportEmissions(gdx)`, where gdx is the location (including the file name) of the fulldata.gdx file of you run.  This will give you a magpie object. I checked, and there are many categories regarding the sources of CO2, CH4, N2O, NO2, and NH3. This is reported in global and regional scales (12 world regions).
